### DISCUSSION TITLE
How can I update the versions in code to be the same as the draft release version?

### DISCUSSION THREAD

**Discussion by justinmchase** [release-drafter/release-drafter#1326] [Q&A]
For example suppose I have a nodejs project which has a `package.json` file with the version `1.0.0`. I now make a PR with a major label which causes a draft release with the version `2.0.0` to be created.

If I were to publish it would create a tag called `2.0.0` but the version in my package.json is still `1.0.0`.

What is the recommended way to keep the version information in my code up to date with the draft version before the Release is published?

**Comment by jetersen** [comment]
`npm version 2.0.0` and then `release`

Take a look here how I do it:
https://github.com/release-drafter/release-drafter/commit/569eb7ee3a85817ab916c8f8ff03a5bd96c9c83e
https://github.com/release-drafter/release-drafter/blob/569eb7ee3a85817ab916c8f8ff03a5bd96c9c83e/package.json#L18
https://github.com/release-drafter/release-drafter/blob/master/.github/workflows/release.yml

**Reply by justinmchase** [reply]
Ok but how are you doing the commit? I keep getting foiled by branch protections, status checks and code owner requirements.

**Reply by jetersen** [reply]
Well your out of luck those are one of the many issues with branch protection 🗡️

I believe nowadays you can add exclusion to the branch protection.
https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/defining-the-mergeability-of-pull-requests/managing-a-branch-protection-rule

**Reply by jetersen** [reply]
You can potentially have a github workflow that does all of the releasing bits 😅
